### PR TITLE
geoserver: wait for DB and DB user to be available

### PR DESCRIPTION
**Pull request by camuffo**
Waiting just for Postgres to be accepting TCP connections is not enough.
The Postgres service is slow to set up during the first deploy. GeoFence would fail to create its tables (as the DB is not ready yet), and stay broken instead of crashing or trying again.

Note that an image including psql is necessary. Here I'm using `governmentpaas/psql` but maybe we'll want to create our own.

### CODE REVIEW

File: charts/geonode/values.yaml [hunk 1]
```diff
@@ -348,9 +348,9 @@ geoserver:
     container_name: geoserver-wait-for-db
     image:
       # init container image
-      name: jwilder/dockerize
+      name: governmentpaas/psql
```
Comment (randomorder) [inline charts/geonode/values.yaml]: can we use an official image instead?

**Review by randomorder — CHANGES_REQUESTED** [review]

File: charts/geonode/values.yaml (hunk 1 above)
Comment (camuffo) [reply charts/geonode/values.yaml]: I wish, but which image is an official image with just psql? I don't want a big image just for psql.

Comment (randomorder) [reply charts/geonode/values.yaml]: use the big image

Comment (randomorder) [reply charts/geonode/values.yaml]: evantually the same used by the zalando cluster if available in the values file

Comment (camuffo) [reply charts/geonode/values.yaml]: Ok. My latest commit uses `postgres:16-alpine`.

**Review by randomorder — APPROVED** [review]

**Pull request merged**